static-instrumenter/maven-plugin/src/main/java/io/opentelemetry/contrib/staticinstrumenter/plugin/maven/ProjectModel.java

```diff
@@ -50,18 +51,42 @@ List<Path> chooseForInstrumentation(@Nullable String artifactName) {
         return new ArrayList<>(Collections.singletonList(artifactFile));
       }
     }
-    String message = "Artifact with name " + artifactName + " not found.";
-    logger.error(message);
```

Comment:
Should we keep that log statement?

Reply:
We don't have to keep it because [an exception with the message will be raised](https://github.com/jeanbisutti/opentelemetry-java-contrib/blob/c1ae5bb1f134094f325c8b1fd0e1e2d796fc6a2e/static-instrumenter/maven-plugin/src/main/java/io/opentelemetry/contrib/staticinstrumenter/plugin/maven/ProjectModel.java#L64) and the plugin execution will stop with this exception.